docs/docs/using-pants/advanced-target-selection.mdx

```diff
@@ -185,3 +185,25 @@ For other goals, you can leverage shell piping to partition the input targets in
 ```bash
 pants list :: | awk 'NR % 5 == 0' | xargs pants package
 ```
+
+## Using CLI aliases
+
+If setting tags on individual targets is not feasible, there are a few other options available to refer to multiple targets.
+
+If you have an operation that you perform often on a certain group of targets, you can use the
+[cli](../../reference/subsystems/cli) subsystem options to create shortcuts. For instance, this alias
+
+```toml title="pants.toml"
+publish-libraries = "--filter-target-type=python_distribution --filter-address-regex=\"['^src/libA/,^src/libB/']\" publish src::"
+```
+
+would you let run `pants publish-libraries` to publish all Python distributions declared in the `src/libA` and `src/libB`
```

Comment:
Suggest moving this so it is connected with the sentence before the code block, to make it a complete sentence not breaken apart by the example. That makes it more fluent to read. I know this is not the style employed in the docs everywhere here.. but I think we should consider adapting it.

```suggestion
would let you run `pants publish-libraries` to publish all Python distributions declared in the `src/libA` and `src/libB`
```

Reply:
> Suggest moving this so it is connected with the sentence before the code block, to make it a complete sentence not breaken apart by the example. 

Sure thing, done!